process/security_baseline.md

```diff
@@ -0,0 +1,252 @@
+  * Sandbox stage - baseline awareness, and initial requirements
+  * Incubation stage - sandbox requirements plus additional baselines to sustain the secure project development and protect early adopters. 
+  * Graduated stage: Incubation baseline plus additional baselines to prepare for its general availability and sustainability. 
+
+This phased approach intends to support maintainers, contributors, and the community in innovating quickly with security built into the design or enabled by default.
+
+### Baseline - Once Sandbox
+| Security Baseline | Objective | How to Implement | How to Verify|
+|-------|-------|-------|-------|
+| A memory-safe language is adopted for new projects or new components. | Reduce  memory safety vulnerabilities at scale. | Choose one of the [memory-safe languages](https://www.memorysafety.org/docs/memory-safety/) <br /> <br />  For preexisting projects in C or C++, follow the [Compiler Options Hardening Guide](https://best.openssf.org/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++) | Check the [file extension](#Standard-File-Extensions-of-Common-Programming-Languages) and compare with the code. <br /> <br /> 
+|Two-factor authentication (2FA) is enabled for repository interactive access. | Reduce the risks of credential compromise and attacks on the digital assets.| 2FA is enabled by default at the enterprise level for all the organizations.<br /> <br /> Refer to [SCM Best Practices](https://best.openssf.org/SCM-BestPractices/) for more information. <br /> <br /> See [instructions for device setup](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication).|[2FA verification, drift detection and correction](#2FA-Verification-Drift-Detection-and-Correction) provides details for verifying 2FA is enabled, monitoring and restoring 2FA if it’s disabled. |
+
+### Baseline - To Become Incubating
+
+As the project codebase grows and more features are added, increasing complexity, it becomes crucial to leverage security tools to identify vulnerabilities in the codebase or dependent software early on. Addressing critical issues early prevents costly fixes in the future. At this stage, projects SHOULD onboard to OpenSSF Scorecard by following the [installation instructions](https://github.com/ossf/scorecard-action#installation) of [Scorecard GitHub Action](https://github.com/ossf/scorecard-action). The Action runs on any repository change and raises alerts. ​​Repository administrators, organization owners, and people with write or maintain access to a repository can view the alerts in the repository’s Security tab. Ensure Scorecard is enabled for the project by following [Scorecard Verify Runs](https://github.com/ossf/scorecard-action?tab=readme-ov-file#verify-runs) instruction.
```

Comment:
This says "projects SHOULD onboard Scorecard", but it seems like there are one or two required Scorecard metrics below. [Current TI incubating requirements](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements-1) already include Scorecard as a MUST, so I think the baseline ought to be consistent with that and also state Scorecard onboarding as a MUST. The table below can then clarify which specific Scorecard metrics are needed for incubating.